src/ValuedTextInput/valuedTextInput.css

```diff
@@ -102,19 +110,48 @@
     }
 }
 
+
 .fakeHovered
 {
+    .container
+    {
+        outline:        0.1rem solid var( --PC-BLUE );
+    }
+
+    .input
+    {
+        outline:        none;
```

Comment:
Since `input` never has an outline this should be set somewhere above, not here.